libs/external-router/driver/testing/src/testing.module.ts

```diff
@@ -0,0 +1,34 @@
+import { NgModule, ModuleWithProviders } from '@angular/core';
+import { CommonModule } from '@angular/common';
+
+import { DaffExternalRouterDriver } from '@daffodil/external-router/driver';
+
+import { DaffExternalRouterTestingDriver } from './testing.service';
+import {
+	DaffExternalRouterDriverTestingConfig,
+	DAFF_EXTERNAL_ROUTER_DRIVER_TESTING_CONFIG,
+} from './config';
+
+@NgModule({
+	declarations: [],
+	imports: [CommonModule],
+})
+export class DaffExternalRouterDriverTestingModule {
+	static forRoot(
+		config: DaffExternalRouterDriverTestingConfig,
```

Comment:
this config type needs to be exposed, so the app dev can type it when they pass it in.

Reply:
https://github.com/graycoreio/daffodil/pull/1240/commits/e84f901398d30c4af371fb257714725339f3640a